package.json

```diff
@@ -13,6 +13,8 @@
 		"lint:swagger": "swagger-spec-validator src/openapi.json",
 		"lint:tsc": "tsc --noEmit -p tsconfig.dev.json",
 		"lint:prettier": "prettier . --check",
+		"lint:sqlfluff": "sqlfluff lint src/database/",
+		"lint:sqlfluff:fix": "sqlfluff fix src/database/",
```

Comment:
I'd expect this to be pat of `format` rather than `lint`

So that would mean we have `format` split into `format:prettier` and `format:sqlfluff` and `format` itself would invoke both.